dashbuilder/modules/dashbuilder/module.yaml

```diff
@@ -25,8 +25,8 @@ ports:
 - value: 8080
 artifacts:
 - name: "rhpam_add_ons_distribution.zip"
-  # rhpam-7.12.0-add-ons.zip
-  md5: "8daf93abb6041073a80b4d4be3f1151d"
+  # rhpam-7.12.1-add-ons.zip
+  md5: "130b988a796987f78e5d3f33770e12d9"
```

Comment:
are `RHPAM_DASHBUILDER_VERSION` and `PRODUCT_VERSION` ok? shouldn't they be `7.12.1`?